<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Software for middle-down Proteomics by middle-down</title>
    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="javascripts/main.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>

      <header>
        <h1>Software for middle-down Proteomics</h1>
        <p></p>
      </header>

      <div id="banner">
        <span id="logo"></span>

        <a href="https://github.com/middle-down/Software" class="button fork"><strong>View On GitHub</strong></a>
        <div class="downloads">
          <span>Downloads:</span>
          <ul>
            <li><a href="https://github.com/middle-down/Software/zipball/master" class="button">ZIP</a></li>
            <li><a href="https://github.com/middle-down/Software/tarball/master" class="button">TAR</a></li>
          </ul>
        </div>
      </div><!-- end banner -->

    <div class="wrapper">
      <nav>
        <ul></ul>
      </nav>
      <section>
        <p><strong>Welcome to the webpage of the middle-down Proteomics software tools.</strong>
The page currently contains <strong>Histone Coder</strong> and <strong>isoScale</strong>, two software produced to validate MS/MS spectra and quantify identified polypeptides by Mascot (Matrix Science, UK) database searching engine. The tools are made in collaboration between the <a href="http://www.sdu.dk">University of Southern Denmark</a> and the <a href="http://www.upenn.edu/">University of Pennsylvania</a></p>

<p>If you want to access our data repository, please visit the <a href="http://crosstalkdb.bmb.sdu.dk/">Cross-talk database</a></p>

<h3>
<a name="histone-coder---download" class="anchor" href="#histone-coder---download"><span class="octicon octicon-link"></span></a>
<strong>Histone Coder</strong> - <a href="https://github.com/middle-down/Software/blob/master/Histone%20Coder.jar?raw=true">Download</a>
</h3>

<p>Histone Coder counts the number of MS/MS ions in a given spectrum to determine the unambiguous localization of a post-translational modification (PTM). The software lists number and type of site determining ions found between the assigned PTM localization by Mascot (Matrix Science) and the closest other amino acids which can host the modification. The PTMs included in the script are phosphorylation (S,T,Y), acetylation (K), mono- and dimethylation (K,R) and trimethylation (K). From the interface it is possible to select filters for spectrum score, fragment ion types and MS/MS tolerance for the search. As additional feature, Histone Coder converts the Mascot output to the standard Brno nomenclature for PTMs (e.g. K4me3K14ac). </p>

<p>Requirements: The software activates with Java. The input is Mascot output in .xml extension. </p>

<h3>
<a name="isoscale---download" class="anchor" href="#isoscale---download"><span class="octicon octicon-link"></span></a>
<strong>isoScale</strong> - <a href="https://github.com/middle-down/Software/blob/master/isoScale-v2.jar?raw=true">Download</a>
</h3>

<p>isoScale is a proteomics software to quantify peptides based on the total ion intensity of the MS/MS spectra. The software provides also relative quantification of isobaric peptides co-fragmented in MS/MS spectra which share the same sequence but have distinct localizations of post-translational modifications (PTMs). The principle of the quantification is described in Pesavento et al. (Analytical Chemistry, 2006). We recommend the use of the software for middle-down or top-down analyses, as it is suitable for direct comparison of single sequences with scrambled combinatorial PTMs. The user interface provides a choice for CID/HCD or ETD fragmentation. It is suitable for high and low resolution data, as it allows the choice of MS/MS ion tolerance for searching the fragment ions required to calculate the fragment ion relative ratio of isobaric peptides (FIRR). </p>

<p>Requirements: The software processes Mascot output files in .csv extension. The software input is the result file, which is the output format of Histone Coder, and a data file, which is the Mascot .csv output </p>

<h3>
<a name="reference" class="anchor" href="#reference"><span class="octicon octicon-link"></span></a><strong>Reference</strong>
</h3>

<p>If you use Histone Coder or isoScale please cite the paper: <strong>Middle-down hybrid chromatography/tandem mass spectrometry workflow for characterization of combinatorial post-translational modifications in histones</strong> (Proteomics, 2014)</p>

<p>Authors:
Simone Sidoli, Veit Schw√§mmle, Chrystian Ruminowicz, Thomas A. Hansen, Xudong Wu, Kristian Helin and Ole N. Jensen</p>

<h3>
<a name="contacts" class="anchor" href="#contacts"><span class="octicon octicon-link"></span></a><strong>Contacts</strong>
</h3>

<p>If having troubles in using the software, or any kind of help, please contact
Simone Sidoli: <a href="mailto:simone.sidoli@gmail.com">simone.sidoli@gmail.com</a></p>
      </section>
      <footer>
        <p>Project maintained by <a href="https://github.com/middle-down">middle-down</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://twitter.com/michigangraham">mattgraham</a></small></p>
      </footer>
    </div>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
    
  </body>
</html>